README.md

```diff
@@ -1,2 +1,35 @@
 # code-gov-site-map-generator
 Generates Sitemap.xml for Code.gov
+
+## Introduction
+
+[Code.gov](https://code.gov) is a website promoting good practices in code development, collaboration, and reuse across the U.S.  Government. Code.gov provides tools and guidance to help agencies implement the [Federal Source Code Policy](https://sourcecode.cio.gov). It also includes an inventory of government custom code to promote reuse between agencies and provides tools to help government and the public collaborate on open source projects.
+
+To learn more about the project, check out this [blog post](https://www.whitehouse.gov/blog/2016/08/08/peoples-code).
+
+Code.gov is an open source project, so we invite your contributions, be it in the form of code, design, or ideas.
+
+## Quick Setup Instructions
+To run this app locally you'll need a code.gov API key.  If you don't have one, go to the following url to obtain one: https://developers.code.gov/key.html
+
+Once you have an API key, clone the repo and run the `npm install` command in the project's root directory to install all of the project’s dependencies.
+
+Run this terminal command: `cp .example.env .env`
+
+Replace `[your api key goes here]` in the `.env` file with your API key
+
+Run the `node index.js` terminal command to run the sitemap generation script.
```

Comment:
Should just use the generate script: `npm run generate`.

Reply:
done